catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/builder/JsonModelBuilderTest.java

```diff
@@ -83,19 +90,19 @@ public void testBinaryLogicTypeAnd() {
 
   @Test
   public void testBinaryLogicTypeAllOperators() {
-    new JsonModelBuilder().beginBinaryLogicType(AND);
-    new JsonModelBuilder().beginBinaryLogicType(OR);
+    new JsonModelBuilder(null).beginBinaryLogicType(AND);
```

Comment:
curious why we pass a null registry for all of these tests.  is there nothing to verify?

Reply:
Nothing to verify for this test. We won't actually interact with the registry at all. We're verifying some other, unrelated functionality of the class. Namely, the mapping and support of our operators. We want to ensure no exceptions are thrown and all operations here are considered valid and supported.